pkg/utils/helm/helm_helper.go

```diff
@@ -345,7 +345,13 @@ func (h *Helper) GetValuesFromChart(repoURL string, chartName string, version st
 	var urls []string
 	for _, chartVersion := range chartVersions {
 		if chartVersion.Version == version {
-			urls = chartVersion.URLs
+			for _, url := range chartVersion.URLs {
+				if !(strings.HasPrefix(url, "https://") || strings.HasPrefix(url, "http://")) {
+					urls = append(urls, fmt.Sprintf("%s/%s", repoURL, url))
```

Comment:
which type of helm repo needs to splice repourl and url? consider to add comments

Reply:
I deploy a harbor according https://github.com/goharbor/harbor/releases, it need to splice repourl and url.